docs/blueprints/EP026.rst

```diff
@@ -0,0 +1,157 @@
+
+
+* **Requirement 7)**
+
+  * Both `MongoDB <https://github.com/mongodb/mongo>`_ and `ArangoDB <https://github.com/arangodb/arangodb>`_ GitHub repositories seem to be well maintained and active with recent releases. MongoDB documentation is more detailed, and MongoDB seems to have a higher number of users.
+
+
+* **Requirement 8)**
+
+  * `pymongo <https://github.com/mongodb/mongo-python-driver>`_ MongoDB's Python official library driver claims that `gevent is supported <https://pymongo.readthedocs.io/en/stable/examples/gevent.html>`_.
+  * `pyarango <https://github.com/ArangoDB-Community/pyArango>`_ official ArangoDB's Python library driver has an `issue <https://github.com/ArangoDB-Community/pyArango/issues/65>`_ reporting a case where incorrect responses where observed in a highly concurrent scenario, so it might have other potential issues and unknowns.
+
+
+* **Requirement 9)**
+
+  * pymongo doesn't support asyncio out of the box, but MongoDB's official `motor project <https://github.com/mongodb/motor>`_ is recommended for asyncio users, and it uses pymongo under the hood. It's great to know that as NApps start to making more use of asyncio this library could be leveraged in the future.
+  * ArangoDB doesn't have an official supported client but there are some project initiatives mentioned `in this thread <https://github.com/ArangoDB-Community/python-arango/issues/95>`_, and it doesn't seem that these initiatives have been battle tested yet.
+
+
+* **Requirement 12)**
+
+  * Elastic APM `supports for pymongo ">=2.9,<3.8" <https://www.elastic.co/guide/en/apm/agent/python/current/supported-technologies.html#automatic-instrumentation-db-mongodb>`_.
+  * Elastic APM doesn't claim to support pyarango nor a custom library instrumented has been found yet.
+
+
+* **Requirement 13)**
+
+  * `MongoDB's documentation <https://docs.mongodb.com/manual/tutorial/backup-and-restore-tools/>`_ is very detailed compared to `Arango's <https://www.arangodb.com/docs/stable/backup-restore.html>`_, both have the tools to dump and restore. It's worth pointing out that ``mongodump`` and ``mongorestore`` `as mentioned here <https://docs.mongodb.com/manual/tutorial/backup-and-restore-tools/#restore-a-database-with-mongorestore>`_ cannot be used in a sharded clusters that have transaction in progress, so temporarily stopping writes might be needed if using these tools, alternatively if a Linux LVM if available, and it can create snapshots, then one can also leverage this mechanism to have point-in-time backups `as documented here <https://docs.mongodb.com/manual/core/backups/#back-up-with-filesystem-snapshots>`_. There are other alternatives, including cloud-based ones, each with their trade-offs. If `Mongo's Ops Manager <https://www.mongodb.com/products/ops-manager>`_ were free to use in production that'd be great.
```

Comment:
```suggestion
  * `MongoDB's documentation <https://docs.mongodb.com/manual/tutorial/backup-and-restore-tools/>`_ is very detailed compared to `Arango's <https://www.arangodb.com/docs/stable/backup-restore.html>`_, both have the tools to dump and restore. It's worth pointing out that ``mongodump`` and ``mongorestore`` `as mentioned here <https://docs.mongodb.com/manual/tutorial/backup-and-restore-tools/#restore-a-database-with-mongorestore>`_ cannot be used in a sharded clusters that have transaction in progress, so temporarily stopping writes might be needed if using these tools, alternatively if a Linux LVM is available, and it can create snapshots, then one can also leverage this mechanism to have point-in-time backups `as documented here <https://docs.mongodb.com/manual/core/backups/#back-up-with-filesystem-snapshots>`_. There are other alternatives, including cloud-based ones, each with their trade-offs. If `Mongo's Ops Manager <https://www.mongodb.com/products/ops-manager>`_ were free to use in production that'd be great.
```